pelican/writers.py

```diff
@@ -139,9 +139,10 @@ def write_file(self, name, template, context, relative_urls=False,
         :param **kwargs: additional variables to pass to the templates
         """
 
-        if name is False or name == "" or\
-           not is_selected_for_writing(self.settings,\
-               os.path.join(self.output_path, name)):
+        if name is False or \
```

Comment:
not sure removing all backslashes is a thing, in this case i opted against it to keep it from producing ))) at the end which would not help readability